app/models/unpublishing.rb

```diff
@@ -89,4 +91,8 @@ def redirect_not_circular
   def ensure_presence_of_content_id
     self.content_id ||= SecureRandom.uuid
   end
+
+  def alternative_uri_matches_gov_uk?
```

Comment:
Would be great if we didn't duplicate this, and call `GovUkUrlFormatValidator.matches_gov_uk?` instead. Would just need to move it above the `private` and prefix it with a `self.`

Reply:
Thanks. I've removed the duplication here.